src/main/kotlin/org/jitsi/nlj/transform/node/incoming/IncomingStatisticsTracker.kt

```diff
@@ -41,20 +41,45 @@ import java.util.concurrent.ConcurrentHashMap
  * Track various statistics about received RTP streams to be used in SR/RR report blocks
  */
 class IncomingStatisticsTracker : ObserverNode("Incoming statistics tracker") {
-    private val streamStats: MutableMap<Long, IncomingStreamStatistics> = ConcurrentHashMap()
+    private val ssrcStats: MutableMap<Long, IncomingSsrcStats> = ConcurrentHashMap()
     private val payloadTypes: MutableMap<Byte, PayloadType> = ConcurrentHashMap()
+
+    /**
+     * The bitrate in bits per seconds (for all RTP streams).
```

Comment:
so this is the sum bitrate of all streams at any given moment, yeah?  maybe it'd be good to rename the variable too. `receiverBitrate` or something?  same for the others below.

Reply:
Same as below, it's already part of IncomingStatisticsSnapshot

Reply:
yeah, sorry i wasn't trying to use receiver to distinguish send from receive, but to distinguish a single stream's value from the entire receiver's values added together (since IncomingStatisticsTracker doesn't really distinguish that either).  maybe there's better name.